examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py

```diff
@@ -163,54 +144,49 @@ def __validate_input_worker_status(self, input_json):
             return False
 
 # ------------------------------------------------------------------------------------------------
-    def __process_worker_set_status(self, worker_id, input_value, response):
+    def WorkerSetStatus(self, **params):
         """
         Function to set the status of worker
         Parameters:
             - worker_id is the worker id specified in the input json str.
-            - input_json_str is a worker request json as per TCF API 5.3.3 Worker Set Status JSON Payload
+            - input_json_str is a worker request json as per TCF API 5.3.3
+                Worker Set Status JSON Payload
             - response is the response object to be returned to client.
```

Comment:
response object in function comment has to be updated with params with this change